worker/buildbot_worker/pbutil.py

```diff
@@ -123,3 +126,19 @@ def failedToGetPerspective(self, why, broker):
 
         # lose the current connection, which will trigger a retry
         broker.transport.loseConnection()
+
+
+def decode(data, encoding='utf-8', errors='strict'):
+    """We need to convert a dictionary where keys and values
+    are bytes, to unicode strings.  This happens when a
+    Python 2 worker sends a dictionary back to a Python 3 master.
```

Comment:
Python2 master sends a dictionarry to python3 worker?

Reply:
Fixed